static/src/javascripts/projects/commercial/modules/hosted/gallery.js

```diff
@@ -317,29 +308,33 @@ class HostedGallery {
         const newIndex = Math.round(progress + 0.75);
         const ctaIndex = HostedGallery.ctaIndex() || -1;
         fastdom.mutate(() => {
-            this.$images.each((image, index) => {
+            this.$images.forEach((image, index) => {
                 const opacity = ((progress - index + 1) * 16) / 11 - 0.0625;
-                bonzo(image).css('opacity', Math.min(Math.max(opacity, 0), 1));
+                image.style.setProperty('opacity', Math.min(Math.max(opacity, 0), 1));
             });
 
-            bonzo(this.$border).css('transform', `rotate(${deg}deg)`);
-            bonzo(this.$border).css('-webkit-transform', `rotate(${deg}deg)`);
+            this.$border.style.setProperty('transform', `rotate(${deg}deg)`);
+            this.$border.style.setProperty('-webkit-transform', `rotate(${deg}deg)`);
```

Comment:
I think we can drop this, there's now over 96% support for the non-prefixed version.

https://caniuse.com/transforms2d

Reply:
Opera mini 😢 ... jk